llvm/include/llvm/IR/DebugProgramInstruction.h

```diff
@@ -426,6 +426,11 @@ class DbgVariableRecord : public DbgRecord, protected DebugValueUser {
   /// Does this describe the address of a local variable. True for dbg.addr
   /// and dbg.declare, but not dbg.value, which describes its value.
   bool isAddressOfVariable() const { return Type == LocationType::Declare; }
+
+  /// Does this describe the value of a local variable. False for dbg.addr
```

Comment:
`llvm.dbg.addr` has been removed from LLVM and replaced with dbg.value(op_deref)